app/get-sommit.js

```diff
@@ -0,0 +1,16 @@
+const { get } = require('./cache')
+
+const getSommit = async (request, key) => {
+  try {
+    const res = await get(request, key)
+    if (res) {
+      console.log('Using existing cache value')
+      return res
+    }
+    return 'mock read through cache method to be created and called'
+  } catch {
+    return undefined
+  }
+}
+
+module.exports = getSommit
```

Comment:
Does this need to be here still or should it be removed before merging?

Reply:
Happy to remove these routes and mock func if you're happy with implementation?